components/authentication/org.wso2.carbon.analytics.msf4j.interceptor.common/src/main/java/org/wso2/carbon/analytics/msf4j/interceptor/common/internal/ServiceComponent.java

```diff
@@ -59,13 +59,12 @@ protected void start(BundleContext bundleContext) throws ConfigurationException
             String exclude = idPClientConfiguration.getProperties().get(IdPClientConstants.EXCLUDE_INTERCEPTOR);
             Boolean isInterceptorEnabled = Boolean.parseBoolean(enableInterceptor);
             DataHolder.getInstance().setInterceptorEnabled(isInterceptorEnabled);
-            if (isInterceptorEnabled) {
-                List<String> collect = Arrays.stream(exclude.replaceAll("\\s*", "")
-                                .split(",")).collect(Collectors.toList());
-                DataHolder.getInstance().setExcludeURLList(collect);
-            } else {
-                DataHolder.getInstance().setExcludeURLList(new ArrayList<>());
+            List<String> excludeURLList = new ArrayList<>();
+            if (exclude != null) {
+                excludeURLList =  Arrays.stream(exclude.replaceAll("\\s*", "")
```

Comment:
Instead of Streams, shall we use `Arrays.asList()` as its much cleaner & shorter

Reply:
Fixed in 0a09e4b